### ISSUE TITLE
[Platform request] Smart TVs and consoles

### ISSUE THREAD

**Issue by matthew-dean** [revery-ui/revery#236]
Cross-platform desktop apps are great, but there's an underserved and growing market for cross-platform apps in the TV space (WebOS, Tizen, Xbox, PS4).

It would be great if you considered adding these to build outputs!

**Comment by wokalski** [comment]
That's true! A performant yet developer friendly framework could be a game changer. Most of those tools are proprietary these days and use C++ and such.

**Comment by bryphe** [comment]
Cool idea @matthew-dean ! I would love to see this happen too 👍

**Comment by jeremyscatigna** [comment]
Would love to help 🙂

**Comment by bryphe** [comment]
@jeremyscatigna - awesome! Is there an additional platform you are particularly interested in, or have experience in?

**Comment by jeremyscatigna** [comment]
AppleTV could be a good one to have too I think @bryphe ! I don't have experiences in this type of platform but I'm pretty interested in learning about it and it could be a really good way ! I'm pretty new to reason too but love the project and would love to learn more about it so nothing better to learn than contributing to a cool project 👍

**Comment by bryphe** [comment]
@jeremyscatigna - cool, AppleTV would be amazing! I unfortunately don't have much experience on that platform either, but would be really interested to see what it takes to get Revery running there.

> I'm pretty new to reason too but love the project and would love to learn more about it so nothing better to learn than contributing to a cool project 👍

Nice, we appreciate the help - and no better way to learn IMO than by building something!

**Comment by jeremyscatigna** [comment]
@bryphe will begin to explore how is it possible to get Revery running there ! Will love any thought or advices of what it could takes :+1
